### DISCUSSION TITLE
CARLA 0.9.14 + SCENARIO RUNNER 0.9.13 - RGB camera not working

### DISCUSSION THREAD

**Discussion by asxja** [carla-simulator/carla#6106] [Q&A]
Hello,

for my purpose I need features which are available in CARLA 0.9.14 (like GBuffer). As there is no SCENARIO RUNNER 0.9.14 available yet, I tried the combination of CARLA 0.9.14 and SR 0.9.13, but I get these kind of RGB camera images:
![image](https://user-images.githubusercontent.com/118812063/212025554-e7aee14a-68b7-4af1-bc6e-8ced49e99619.png)

Will this be fixed with a new SCENARIO RUNNER or can you help me to solve it please? When will SCENARIO RUNNER 0.9.14 be available?

Thank you
Andreas

**Comment by Niko-Stark** [comment]
This problem may be caused by the width of the image. the image_width shoule be K*64, and K is a integer.
